core-services/prow/02_config/_config.yaml

```diff
@@ -2352,6 +2352,7 @@ tide:
     - openshift-priv/origin
     - openshift/network-tools
     - openshift/origin
+    - openshift/cluster-cloud-controller-manager-operator
```

Comment:
This placement does not look correct. Is this repo supposed to follow the "standard" OCP release cycle and merge criteria (needing bugs etc.)

Reply:
Ah I see, this was created by `repo-init`, right? We need to fix that, piggybacking on origin no longer works :)

Reply:
Can you please remove the repo from the two queries that the tool added, and add it to the *three* queries that contain the `openshift/cluster-version-operator` repo? You do not need to bother with sorting, just add it anywhere in the list and run `make prow-config` and it will handle the ordering for you.

Reply:
https://github.com/openshift/ci-tools/pull/1606 will fix this for others

Reply:
yeah, thank you for the clarification! I updated the patch as you asked